sql/src/main/java/org/apache/druid/sql/calcite/aggregation/builtin/EarliestLatestAnySqlAggregator.java

```diff
@@ -313,12 +328,48 @@ public RelDataType inferReturnType(SqlOperatorBinding sqlOperatorBinding)
     }
   }
 
+  private static class TimeColIdentifer extends SqlIdentifier
+  {
+
+    public TimeColIdentifer()
+    {
+      super("__time", SqlParserPos.ZERO);
+    }
+
+    @Override
+    public <R> R accept(SqlVisitor<R> visitor)
+    {
+
+      try {
+        return super.accept(visitor);
+      }
+      catch (CalciteContextException e) {
+        if (e.getCause() instanceof SqlValidatorException) {
+          throw DruidException.forPersona(DruidException.Persona.ADMIN)
```

Comment:
```suggestion
          throw DruidException.forPersona(DruidException.Persona.USER)
```

Reply:
Chose `ADMIN` Persona here since the exceptions in existing tests such as `testTimeColumnAggregationsOnLookups` had  had the same -- not sure why though. If that's incorrect, shall update the tests as well.

Reply:
Perhaps here's the reason you are looking for: https://github.com/apache/druid/blob/master/sql/src/main/java/org/apache/druid/sql/calcite/planner/QueryHandler.java#L692

However, in this case, it seems like we do know the exact reason why the query failed - It's due to the absence of a time column. So perhaps change the wording to be more assertive.

Reply:
I'm not sure if the reason will always be absence of time column - what if there's a case of a join where `__time` column is coming from both the tables, in that case I think it could also be an `ambiguous` column.

Reply:
I don't know what will admin do with these planning errors even if they are hints. The hints are meant for the end-user. I don't want to hold this PR so would let it merge as it is.